packages/api/src/platforms/vtex/resolvers/validateCart.ts

```diff
@@ -219,20 +219,23 @@ async function getOrderNumberFromSession(
 }
 
 // Returns the regionalized orderForm
-const getOrderForm = async (
-  id: string,
-  session: Maybe<IStoreSession> | undefined,
-  { clients: { commerce } }: Context
-) => {
-  const orderForm = await commerce.checkout.orderForm({
+const getOrderForm = async (id: string, { clients: { commerce } }: Context) => {
+  return await commerce.checkout.orderForm({
```

Comment:
in this case async and await are redundant, you can return directly
```suggestion
const getOrderForm = async (id: string, { clients: { commerce } }: Context) => {
  return commerce.checkout.orderForm({
```